Make.inc

```diff
@@ -82,7 +82,7 @@ endif
 USE_OPROFILE_JITEVENTS ?= 0
 
 # Set to 1 to enable profiling with perf
-USE_PERF_JITEVENTS ?= 0
+USE_PERF_JITEVENTS ?= 1
```

Comment:
We should have made this conditional on `ifeq ($(OS),Linux)`, since LLVM now refuses to build on non-linux

Reply:
Mea culpa, https://github.com/JuliaPackaging/Yggdrasil/blob/80d832c7c02bcb52e787a883ee312e1d8eae223a/L/LLVM/common.jl#L150

Will push a fix.